package.json

```diff
@@ -88,11 +89,11 @@
     "test:systemd": "docker run -v $(pwd):/tmp/pump.io:ro -v $NVM_DIR:/.nvm:ro ubuntu sh -c \"export NVM_DIR=$NVM_DIR; . $NVM_DIR/nvm.sh; nvm use $TRAVIS_NODE_VERSION; cd /tmp/pump.io; npm run test:systemd:run\"",
     "test:systemd:run": "mkdir -p /tmp/pump.io/uploads; sed 's/%i.service//g' pump.io@.service | sudo tee /etc/systemd/system/pump.io.service; sudo systemctl daemon-reload; sudo systemctl start pump.io.service; npm run test:system",
     "test:report-coverage": "lcov-result-merger 'coverage*/lcov.info' | coveralls",
-    "lint": "npm run lint:jshint && npm run lint:jscs && npm run lint:jade && npm run lint:json && npm run lint:deps",
-    "lint:deps": "dependency-check package.json bin/* test/*.js > /dev/null && dependency-check --unused package.json -i dependency-check -i jscs -i jshint -i jsonlint -i pug-lint -i nyc -i lcov-result-merger -i coveralls bin/* test/*.js > /dev/null",
+    "lint": "npm run lint:jshint && npm run lint:jscs && npm run lint:pug && npm run lint:json && npm run lint:deps",
+    "lint:deps": "dependency-check package.json bin/* test/*.js > /dev/null && dependency-check --unused package.json -i dependency-check -i jscs -i jshint -i jsonlint -i pug-cli -i pug-lint -i nyc -i lcov-result-merger -i coveralls bin/* test/*.js > /dev/null",
     "lint:jshint": "jshint --exclude 'public/javascript/libs' bin/* lib routes public/javascript test",
     "lint:jscs": "jscs bin/* lib public/javascript/pump* routes test",
-    "lint:jade": "pug-lint public/template/*.jade",
```

Comment:
It would be nice to keep something here for muscle memory backwards-compatibility; something like:

    "lint:jade": "npm run lint:pug && echo 'use `npm run lint:pug` next time' 1>&2",

Reply:
done https://github.com/pump-io/pump.io/pull/1580/commits/cadad9056cf2f254a57b04180f48d001f3acfb70